tests/installation/ipxe_install.pm

```diff
@@ -17,28 +17,13 @@ use warnings;
 use utils;
 use testapi;
 use bmwqemu;
+use ipmi_backend_utils;
 
 use HTTP::Tiny;
 use IPC::Run;
 use Socket;
 use Time::HiRes 'sleep';
 
-sub ipmitool {
-    my ($cmd) = @_;
-
-    my @cmd = ('ipmitool', '-I', 'lanplus', '-H', $bmwqemu::vars{IPMI_HOSTNAME}, '-U', $bmwqemu::vars{IPMI_USER}, '-P', $bmwqemu::vars{IPMI_PASSWORD});
-    push(@cmd, split(/ /, $cmd));
-
-    my ($stdin, $stdout, $stderr, $ret);
-    print @cmd;
-    $ret = IPC::Run::run(\@cmd, \$stdin, \$stdout, \$stderr);
-    chomp $stdout;
-    chomp $stderr;
-
-    die join(' ', @cmd) . ": $stderr" unless ($ret);
-    bmwqemu::diag("IPMI: $stdout");
-    return $stdout;
-}
```

Comment:
This was just duplication of code from `ipmi_backend_utils` ?

Reply:
Yes, so I just discovered the duplication and removed it.